cli/Releasing.md

```diff
@@ -0,0 +1,129 @@
+- All PRs created by the release script should be edited to clarify that they are temporary and will be deleted when testing is finished.
+
+### Release checklist template differences
+
+The release checklist is not used for alpha releases. When testing is finished, please close all PRs and delete all branches created by the script.
+
+## 2. Regular
+
+### When
+
+On Thursdays, the week before the main apps (WPiOS & WPAndroid) have cut their releases, every 2 weeks.
+
+### Branches
+
+For example when releasing gutenberg-mobile `1.11.0`.
+
+| Repo             | Cut From | Branch Name                        | Merging To      |
+| ---------------- | -------- | ---------------------------------- | --------------- |
+| gutenberg        | trunk    | rnmobile/release_1.11.0            | trunk           |
+| gutenberg-mobile | trunk    | release/1.11.0                     | trunk           |
+| WPAndroid        | trunk    | gutenberg/integrate_release_1.11.0 | trunk           |
+| WPiOS            | trunk    | gutenberg/integrate_release_1.11.0 | trunk           |
+
+## 3. Betafix
+
+### When
+
+A fix is targeting a main app version that is not yet released (meaning the release branch is cut but it's still in beta) and a new gutenberg-mobile release is needed.
+
+### Branches
+
+For example when releasing gutenberg-mobile `1.11.1` while main apps version `22.2.0` is in beta which currently has gutenberg-mobile `1.11.0` in it.
+At the same time there could also be a regular release going on for example for gutenberg-mobile version `1.12.0`.
+
+| Repo             | Cut From                | Branch Name                        | Merging To                                                       |
+| ---------------- | ----------------------- | ---------------------------------- | ---------------------------------------------------------------- |
+| gutenberg        | rnmobile/release_1.11.0 (tag) | rnmobile/release_1.11.1            | trunk & (maybe also) rnmobile/release_1.12.0                     |
+| gutenberg-mobile | v1.11.0 (tag)          | release/1.11.1                     | trunk & (maybe also) release/1.12.0                              |
+| WPAndroid        | release/22.2.0          | gutenberg/integrate_release_1.11.1 | release/22.2.0 & (maybe also) gutenberg/integrate_release_1.12.0 |
+| WPiOS            | release/22.2.0          | gutenberg/integrate_release_1.11.1 | release/22.2.0 & (maybe also) gutenberg/integrate_release_1.12.0 |
+
+### Automation script differences
+
+1. Before running the script switch to the relevant branch to cut from in gutenberg-mobile repo.
+1. Run the CLI tool as normal
+1. When asked by the script enter the relevant branch names to cut from (to target) in other repos.
+1. If a commit that is fixing the issue is already merged to gutenberg, when asked by the script enter the commit hash to be cherry-picked.
```

Comment:
This is still WIP but the CLI will accept a list of PRs via a pr flag ( `--prs` ). I Was considering an interactive prompt but we usually only include a single pr in a patch release I figure we don't need the prompt.